src/slot_machine.c

```diff
@@ -2178,83 +2378,105 @@ static bool8 AreTagsAtPosition_Reel1(s16 pos, u8 tag1, u8 tag2)
     return FALSE;
 }
 
-static bool8 AreCherriesOnScreen_Reel1(s16 offsetFromCenter)
+// Returns true if there are cherries on screen in reel 1 after the given number
+// of turns.
+static bool8 AreCherriesOnScreen_Reel1(s16 turns)
 {
-    if (GetTag(LEFT_REEL, 1 - offsetFromCenter) == GFXTAG_CHERRY
-     || GetTag(LEFT_REEL, 2 - offsetFromCenter) == GFXTAG_CHERRY
-     || GetTag(LEFT_REEL, 3 - offsetFromCenter) == GFXTAG_CHERRY)
+    if (GetTag(LEFT_REEL, 1 - turns) == GFXTAG_CHERRY ||
+        GetTag(LEFT_REEL, 2 - turns) == GFXTAG_CHERRY ||
+        GetTag(LEFT_REEL, 3 - turns) == GFXTAG_CHERRY)
```

Comment:
Moving the `||` to the end of the line here is generally inconsistent with how the repo is formatted (16 uses at the end vs 550 at the start)

Reply:
Reverted